csatar-theme/pages/letoltes.htm

```diff
@@ -0,0 +1,28 @@
+url = "/letoltes/:extension/:file"
+layout = "default"
+title = "Csv letöltés"
```

Comment:
Szerintem ez a szoveg latszik a tab-on a bongeszoben. Gondolom, ide is jo lenne beirni az xlsx-t. - 1*